src/components/section-navigation/_section-navigation.scss

```diff
@@ -45,6 +44,8 @@
 
     &__item-header {
         @extend .ons-u-mb-no;
+
```

Comment:
```suggestion
```
Would be a bit neater without this line but thats a minimal thing. Looks good other than that